### PR TITLE
Test(web-twig): Slightly improve readability of Twig snapshots by adding more line breaks

### PR DESCRIPTION
**Pull request by adamkudrna**
This is pretty much everything I was able to find. Not great, but slightly better.

https://api.html-tidy.org/tidy/quickref_5.8.0.html#vertical-space

### CODE REVIEW

**Review by crishpeen — APPROVED** [review]

File: packages/web-twig/tests/ComponentSnapshotTest.php [hunk 1]
```diff
@@ -61,6 +61,7 @@ private function formatHtml(string $html): string
         $config = [
             'indent' => true,
             'output-xhtml' => true,
+            'vertical-space' => true,
```
Comment (literat) [inline packages/web-twig/tests/ComponentSnapshotTest.php:64]: Try adding also `indent-attributes`...

Comment (literat) [reply packages/web-twig/tests/ComponentSnapshotTest.php:64]: https://api.html-tidy.org/tidy/quickref_5.8.0.html#indent-attributes

**Review by literat — APPROVED** [review]

File: packages/web-twig/tests/ComponentSnapshotTest.php (hunk 1 above)
Comment (adamkudrna) [reply packages/web-twig/tests/ComponentSnapshotTest.php:64]: I tried, but didn't see any result… I'll try one more time 🙂.

Comment (adamkudrna) [reply packages/web-twig/tests/ComponentSnapshotTest.php:64]: Oh, even worse: it joins some lines into one…

image

I wouldn't do that 🙂.

**Pull request merged**